sql/cvx_codes.sql

```diff
@@ -142,7 +142,64 @@ VALUES
+(NULL, "Yellow fever vaccine alternative formulation", "Yellow fever vaccine - alt", 183, 100, '', 0, 0, '', '', '', 1),
+(NULL, "Yellow fever vaccine, unspecified formulation", "Yellow fever, unspecified formulation", 184, 100, '', 0, 0, '', '', '', 1),
+(NULL, "Seasonal, quadrivalent, recombinant, injectable influenza vaccine, preservative free", "influenza, recombinant, quadrivalent,injectable, preservative free", 185, 100, '', 0, 0, '', '', '', 1),
+(NULL, "Influenza, injectable, Madin Darby Canine Kidney, quadrivalent with preservative", "Influenza, injectable, MDCK, quadrivalent, preservative", 186, 100, '', 0, 0, '', '', '', 1),
+(NULL, "zoster vaccine recombinant", "zoster recombinant", 187, 100, '', 0, 0, '', '', '', 1),
+(NULL, "zoster vaccine, unspecified formulation", "zoster, unspecified formulation", 188, 100, '', 0, 0, '', '', '', 1),
+(NULL, "Hepatitis B vaccine (recombinant), CpG adjuvanted", "HepB-CpG", 189, 100, '', 0, 0, '', '', '', 1),
+(NULL, "Typhoid conjugate vaccine (non-US)", "Typhoid conjugate vaccine (TCV)", 190, 100, '', 0, 0, '', '', '', 1),
+(NULL, "meningococcal A polysaccharide vaccine (non-US)", "meningococcal A polysaccharide (non-US)", 191, 100, '', 0, 0, '', '', '', 1),
+(NULL, "meningococcal AC polysaccharide vaccine (non-US)", "meningococcal AC polysaccharide (non-US)", 192, 100, '', 0, 0, '', '', '', 1),
+(NULL, "hepatitis A and hepatitis B vaccine, pediatric/adolescent (non-US)", "Hep A-Hep B, pediatric/adolescent", 193, 100, '', 0, 0, '', '', '', 1),
+(NULL, "Influenza, Southern Hemisphere, unspecified formulation (Non-US)", "Influenza, Southern Hemisphere", 194, 100, '', 0, 0, '', '', '', 1),
+(NULL, "Diphtheria, Tetanus, Poliomyelitis adsorbed", "DT, IPV adsorbed", 195, 100, '', 0, 0, '', '', '', 1),
+(NULL, "tetanus and diphtheria toxoids, adsorbed, preservative free, for adult use, Lf unspecified", "Td, adsorbed, preservative free, adult use, Lf unspecified", 196, 100, '', 0, 0, '', '', '', 1),
+(NULL, "influenza, high-dose seasonal, quadrivalent, preservative free", "influenza, high-dose, quadrivalent", 197, 100, '', 0, 0, '', '', '', 1),
+(NULL, "Diphtheria, pertussis, tetanus, hepatitis B, Haemophilus Influenza Type b, (Pentavalent)", "DTP-hepB-Hib Pentavalent Non-US", 198, 100, '', 0, 0, '', '', '', 1),
+(NULL, "influenza, seasonal, Southern Hemisphere, quadrivalent, pediatric 0.25mL dose, preservative free", "influenza, Southern Hemisphere, pediatric, preservative free", 200, 100, '', 0, 0, '', '', '', 1),
+(NULL, "influenza, seasonal, Southern Hemisphere, quadrivalent, 0.5mL dose, no preservative", "influenza, Southern Hemisphere, preservative free", 201, 100, '', 0, 0, '', '', '', 1),
+(NULL, "influenza, seasonal, Southern Hemisphere, quadrivalent, 0.5mL dose, with preservative", "influenza, Southern Hemisphere, quadrivalent, with preservative", 202, 100, '', 0, 0, '', '', '', 1),
+(NULL, "AS03 Adjuvant", "AS03 Adjuvant", 801, 100, '', 0, 0, '', '', '', 1),
```

Comment:
@stephenwaite , thanks for taking on this joyous task :)
The new ones will need to also be added in the upgrade script